packages/framework/azure-client/src/AzureClient.ts

```diff
@@ -128,6 +169,43 @@ export class AzureClient {
         return { container: fluidContainer, services };
     }
 
+    /**
+     * Get the list of versions for specific container
+     * @param id - Unique ID of the source container in Azure Fluid Relay.
+     * @param maxCount - Max number of versions to retreive,
+     * @returns Array of available versions
+     */
+     public async getContainerVersions(
+        id: string,
+        maxCount: number,
```

Comment:
This seems like the right place to put this. Did you consider any other options that didn't work?

Reply:
I have folded `maxCount` under `AzureGetVersionsOptions`. More on that in the other discussion,